packages/EasyCore/src/Doctrine/Events/FixPostgreSqlDefaultSchemaListener.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Doctrine\Events;
+
+use Doctrine\DBAL\Schema\PostgreSqlSchemaManager;
+use Doctrine\ORM\Tools\Event\GenerateSchemaEventArgs;
+
+/**
+ * Workaround for an issue: https://github.com/doctrine/dbal/issues/1110.
+ *
+ * @codeCoverageIgnore
```

Comment:
Please add an `@author https://github.com/vudaltsov` annotation to mention the original author from the issue above.

Reply:
Done.